### DISCUSSION TITLE
Bug report: camera inside symbol

### DISCUSSION THREAD

**Discussion by overcesium** [keiwando/toonsquid#378] [General]
I have a symbol with a camera inside it. I wanted to try converting that camera in a symbol, so now I have a symbol with a camera included (B) inside a symbol (A). When I watch from A the camera zoom animation I made in B, the animation and the camera doesn't work, it displays like normal size symbol. Now whenever I try to go back to the main timeline, the app crashes. Also, on that specific file, I cannot use the camera anymore, it doesn't show anymore as an available option in the timeline "+" menú. This happens on both my ipad pro 2016 and 2022.

**Comment by keiwando** [comment]
If you could please send me the project file to contact@toonsquid.com I can take a look at what exactly is causing the crash and hopefully even fix the project file itself on top of fixing the underlying bug in a future update.

**Comment by keiwando** [comment]
This should now be fixed with the 1.9.4 update. Thanks for reporting it!
